install_config/registry/extended_registry_configuration.adoc

```diff
@@ -589,32 +589,31 @@ configuration reference] above for particular option.
 [[docker-registry-configuration-reference-openshift]]
 === OpenShift
 
-This section reviews the configuration of global settings related to
-{product-title} specific features. In the future, `openshift`-related settings
-in the xref:docker-registry-configuration-reference-middleware[middleware
-section] will be obsolete. Currently, this section allows you to configure only
-for metrics collection.
+This section reviews the configuration of global settings for features specific
+to {product-title}. In a future release, `openshift`-related settings in the
+xref:docker-registry-configuration-reference-middleware[Middleware] section will
+be obsoleted.
+
+Currently, this section only allows you to configure registry metrics
+collection:
```

Comment:
Do we want to remove _only_? Seems negative. 
"Currently, this section allows you to..." seems OK.